### PR TITLE
overlapping marker spiderfier

### PR DESCRIPTION
**Pull request by emre2038**
## Description
making it easier for the users to click on markers that overlap with each other

## How to test it locally
* select a theme from the service mapper
* wait until the markers are created
* click on a group of markers
* confirm that the marker that is clicked is now separated from the others

## Screenshots
image
image

### Updated
* `point_data.js` to use `overlapping-marker-spiderfier-leaflet` library

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally
- [x] 👩‍🎨 does the design matches the [Demo](https://wazimap-ng-v1.webflow.io/demo)

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [ ]  🔖 issue linked
- [x]  📖 changelog filled out
- [x] commit messages are meaningful

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers

### Testing

- [ ]  ✅ added (appropriate) unit tests
- [ ]  💢 edge cases in tests were considered
- [x]  ✅ ran tests locally & are passing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "leaflet.markercluster": "^1.4.1",
     "main": "^1000.0.1",
     "main.js": "0.0.1",
+    "overlapping-marker-spiderfier-leaflet": "https://github.com/emre2038/OverlappingMarkerSpiderfier-Leaflet.git",
```
Comment (jbothma) [inline package.json:28]: pin to a commit with #

Comment (jbothma) [inline package.json:28]: Use a fork on an open up repo

Comment (emre2038) [reply package.json]: done

Comment (emre2038) [reply package.json]: done

**Pull request closed without merging**